cpp/doc/source/conf.py

```diff
@@ -17,25 +17,26 @@
 
 # -- Project information -----------------------------------------------------
 
-project = 'DOLFINx'
-copyright = '2022, FEniCS Project'
-author = 'FEniCS Project'
+project = "DOLFINx"
+copyright = "2022, FEniCS Project"
+author = "FEniCS Project"
 
 # The full version, including alpha/beta/rc tags
-release = '0.3.1'
+release = "0.3.1"
```

Comment:
This number seems a bit out of date :P

Reply:
(of course that's unrelated to these changes, but noticed it while scrolling)

Reply:
I think we should leave this PR as minimal as possible with respect to it's goal of introducing ruff formatting.